components/brave_private_cdn/private_cdn_helper_unittest.cc

```diff
@@ -5,77 +5,77 @@
 
 #include "brave/components/brave_private_cdn/private_cdn_helper.h"
 
+#include <array>
 #include <string>
 #include <string_view>
 
 #include "base/compiler_specific.h"
+#include "base/types/zip.h"
 #include "testing/gtest/include/gtest/gtest.h"
 
-TEST(BravePrivateCdnHelper, RemovePadding) {
+TEST(BravePrivateCdnHelperTest, RemovePadding) {
   using std::string_literals::operator""s;
```

Comment:
nit: this should be removed if we aren't using it

Reply:
Actually I've changed it to the `string_view` literal.